tests/core/test_cli_entry.py

```diff
@@ -27,9 +27,15 @@ def tearDown(self):
         if path.exists(status_file_path):
             os.remove(status_file_path)
 
+        # Clean up the logger in the case where a test fails to complete
+        print("Cleaning up directory logger")
+        if logger.directory_logger:
+            logger.remove_directory_logger()
```

Comment:
Small bug fix, when multiple tests run, and one fails, the following tests were failing too because the directory logger was still active.